docs/apps/interchain-accounts/legacy/integration.md

```diff
@@ -0,0 +1,193 @@
+<!--
+order: 1
+-->
+
+# Integration
```

Comment:
were modifications made to the legacy files that need to be reviewed?

Reply:
I think the only changes I made were in the first paragraphs of each page, maybe to clarify the legacy idea a bit more. But the rest of the content (pseudocode, etc) I think it's the same as before.